modules/codemod/lib/v9/index.ts

```diff
@@ -1,12 +1,16 @@
 import {Transform} from 'jscodeshift';
 
 import softDeprecateStack from './deprecateStack';
+import promoteToast from './promoteToast';
+import compoundToast from './compoundToast';
 
 const transform: Transform = (file, api, options) => {
   // These will run in order. If your transform depends on others, place yours after dependent transforms
   const fixes = [
     // add codemods here
     softDeprecateStack,
+    promoteToast,
+    compoundToast,
```

Comment:
I broke out the codemod in two steps for easier codemod writing. I first promote the component, then I convert it to a compound component